challengeFinal/Flow/Story/Coordinator/StoryCoordinator.swift

```diff
@@ -55,9 +57,19 @@ public class StoryCoordinator: Coordinator {
     }
 
     func showChoices(with childNodes: [StoryNode]) {
+        var dynamics: [DynamicTypes]
+        var selectedDynamic: DynamicTypes
```

Comment:
What you think about passing the player points to ChoiceViewInfosObject. Then, the model sets the dynamics variables. In that way, we avoid creating view logic in the Coordinator.